runtime/tools/spec-builder/test/utils/spawn-child-process.js

```diff
@@ -0,0 +1,47 @@
+const { spawn } = require('child_process');
+
+function buildSpec(file, output) {
+    return new Promise(function(resolve, reject) {
+        const defaults = {
+            cwd: process.cwd(),
+            env: process.env
+        }
+        const command = spawn('node', ['index.js', '-f', file, '-o', output], defaults);
+    
+        console.log(`generating raw json file ${output} ...`);
```

Comment:
normally when you have tests you are not logging anything other than the test results (except when you are developing/debugging smth), so I think that it will be better to remove the console log